README.md

```diff
@@ -1,28 +1,28 @@
 # CodeCarbon
 While computing currently represents roughly 0.5% of the world’s energy consumption, that percentage is projected to grow beyond 2% in the coming years, which will entail a significant rise in global CO2 emissions if not done properly. Given this increase, it is important to quantify and track the extent and origin of this energy usage, and to minimize the emissions incurred as much as possible.
 
-For this purpose, we created **CodeCarbon**, a Python package for tracking the carbon emissions produced by various kinds of computer programs, from straightforward algorithms to deep neural networks. 
+For this purpose, we created **CodeCarbon**, a Python package for tracking the carbon emissions produced by various kinds of computer programs, from straightforward algorithms to deep neural networks.
 
-By taking into account your computing infrastructure, location, usage and running time, CodeCarbon can provide an estimate of how much CO<sub>2</sub> you produced, and give you some comparisons with common modes of transporation to give you an order of magnitude. 
+By taking into account your computing infrastructure, location, usage and running time, CodeCarbon can provide an estimate of how much CO<sub>2</sub> you produced, and give you some comparisons with common modes of transporation to give you an order of magnitude.
 
 Our hope is that this package will be used widely for estimating the carbon footprint of computing, and for establishing best practices with regards to the disclosure and reduction of this footprint.
 
 Follow the steps below to set up the package and don't hesitate to open an issue if you need help!
 
 
 ## Setup
-Create a virtual environment using `conda` for easier management of dependencies and packages. 
+Create a virtual environment using `conda` for easier management of dependencies and packages.
 For installing conda, follow the instructions on the [official conda website](https://docs.conda.io/projects/conda/en/latest/user-guide/install/)
 
 ```
 conda create --name codecarbon python=3.6
 conda activate codecarbon
-pip install . 
+pip install .
 ```
 
 `codecarbon` will now be installed in your the local environment
```

Comment:
Do you think this is confusing to have the conda environment also called "codecarbon"?

> `codecarbon` will now be installed in your the local environment

Since it has the same name, that sentence can be confusiong too.

Reply:
Yeah once we distribute this as a pip package can we change the installation guidelines to `pip install codecarbon`?

Reply:
I agree, I always put "env" somewhere in my env names not to confuse them with anything else